hal/src/main/java/edu/wpi/first/hal/sim/DriverStationSim.java

```diff
@@ -78,6 +79,12 @@ public void setDsAttached(boolean dsAttached) {
   public void notifyNewData() {
     DriverStationDataJNI.notifyNewData();
   }
+  public void waitForCachedControlData() {
```

Comment:
These need to be calling the new cached methods. Theyre separate from the current ones.